apps/explorer/src/components/Layout/index.tsx

```diff
@@ -1,81 +1,37 @@
     return (
-        // NOTE: We set a top-level key here to force the entire react tree to be re-created when the network changes:
-        <Fragment key={network}>
-            <WalletKitProvider
-                /*autoConnect={false}*/
-                enableUnsafeBurner={import.meta.env.DEV}
-            >
-                <GrowthBookProvider growthbook={growthbook}>
-                    <QueryClientProvider client={queryClient}>
-                        <RpcClientContext.Provider value={jsonRpcProvider}>
-                            <NetworkContext.Provider
-                                value={[network, setNetwork]}
-                            >
-                                <div className="w-full">
-                                    <Header />
-                                    <main className="relative z-10 min-h-screen bg-offwhite">
-                                        <section className="mx-auto max-w-[1440px] px-5 py-10 2xl:px-0">
-                                            <Outlet />
-                                        </section>
-                                    </main>
-                                    <Footer />
-                                </div>
+        <GrowthBookProvider growthbook={growthbook}>
+            <WithPostHogMaybeEnabled>
+                <LayoutContent />
+            </WithPostHogMaybeEnabled>
+        </GrowthBookProvider>
+    );
+}
 
-                                <Toaster
-                                    position="bottom-center"
-                                    gutter={8}
-                                    containerStyle={{
-                                        top: 40,
-                                        left: 40,
-                                        bottom: 40,
-                                        right: 40,
-                                    }}
-                                    toastOptions={{
-                                        duration: 4000,
-                                        success: {
-                                            icon: null,
-                                            className:
-                                                '!bg-success-light !text-success-dark',
-                                        },
-                                        error: {
-                                            icon: null,
-                                            className:
-                                                '!bg-issue-light !text-issue-dark',
-                                        },
-                                    }}
-                                />
-                                <ReactQueryDevtools />
-                            </NetworkContext.Provider>
-                        </RpcClientContext.Provider>
-                    </QueryClientProvider>
-                </GrowthBookProvider>
-            </WalletKitProvider>
-        </Fragment>
+type WithPostHogEnabledProps = {
```

Comment:
the way I've set this up is maybe a little ugly, but this will all be re-factored depending on whether or not we keep PostHog and instrument it in our wallet